docs/src/pages/[platform]/components/authenticator/customization/LabelsAndTextDemo.tsx

```diff
@@ -67,37 +62,19 @@ function Screen({ Component }: ScreenProps) {
     );
   }
 
-  return <Component key={version} />;
+  return Component;
 }
 
 export function LabelsAndTextDemo({ Component }: ScreenProps) {
-  const OnMachineInit = ({ children }) => {
-    /**
-     * This waits for Authenticator machine to init before its inner components
-     * start consuming machine context.
-     */
-    const { route, _send } = useAuthenticator();
-    React.useEffect(() => {
-      if (route === 'idle') {
-        _send('INIT', { data: {} });
-      }
-    }, []);
-    if (!route || route === 'idle' || route === 'setup') return null;
-
-    return <>{children}</>;
-  };
-
   return (
     <Authenticator.Provider>
-      <OnMachineInit>
-        <View data-amplify-authenticator="">
-          <View data-amplify-container="">
-            <View data-amplify-body>
-              <Screen Component={Component} />
-            </View>
+      <View data-amplify-authenticator="">
+        <View data-amplify-container="">
+          <View data-amplify-body>
+            <Screen Component={Component} />
```

Comment:
This used to accept a `Authenticator.<someComponent>`. However, since our refactor we've updated the `Authenticator.tsx` file and now *almost* all components require a `CustomeComponentsContext.Provider` hook. Which is not normally exported outside of the internal React implementation.

Instead, I'm using the built-in `initialState` props for the `Authenticator`, which is closer to what an actual user would use.

The limitation of this is that we no longer will be able to show the `Authenticator.SetupTOTP` example page. I feel like this is an OK tradeoff, since the users can still see the example text, they just won't see a demo of it.

Reply:
I like this, this example was very flimsy.